crates/node/src/interactive_cli/call.rs

```diff
@@ -22,6 +22,8 @@ pub struct CallCommand {
     /// The identity of the executor
     #[clap(long = "as", default_value = "default")]
     executor: Alias<PublicKey>,
+    #[clap(long = "substitute")]
+    substitutes: Vec<Alias<PublicKey>>,
```

Comment:
after settling on the design from `meroctl`, you can apply them here too.. with doc-comments and everything

Reply:
we're still missing these